src/main/webapp/creatives/maps/dashboardScript.js

```diff
@@ -1,23 +1,30 @@
-
-import {URL} from './utils.js'
-
 google.charts.load('current', {'packages':['corechart']});
 google.charts.setOnLoadCallback(drawChart);
 
+const BASE_URL = "http://step-capstone-team33-2020.appspot.com"
+
 /** Fetches interaction data from server and uses it to create a chart. */
-function drawChart() {
-  fetch(`/${URL.DASHBOARD}`).then(response => response.json())
+function drawChart(parameter = "") {
+  fetch(`${BASE_URL}/dashboard?${parameter}`, 
+  {
+    headers : { 
+      'Content-Type': 'application/json',
+      'Accept': 'application/json'
+    },
+    mode:'no-cors',
```

Comment:
Did you confirm that no-cors is definately needed?  IN production, since this (should?) be a same-origin request, I don't think it should be needed.
If it doesn't work without though, fine to leave in.

Reply:
altered code with kaleab and now got rid of it